src/pkg/services/m365/api/events.go

```diff
@@ -748,5 +753,5 @@ func EventCollisionKey(item models.Eventable) string {
 
 	// this result gets hashed to ensure that an enormous list of attendees
 	// doesn't generate a multi-kb collision key.
-	return clues.ConcealWith(clues.SHA256, subject+a+t+s+e+r)
+	return subject + t + s + e + r + strconv.FormatBool(draft) + strconv.FormatBool(cancelled)
```

Comment:
these need better names. It's way too cryptic to be easily readable so it's difficult to actually determine what's happening and if the code is doing what we think it's doing